addons/mail/static/src/components/thread_view/thread_view_tests.js

```diff
@@ -241,6 +241,57 @@ QUnit.test('message list asc order', async function (assert) {
     );
 });
 
+QUnit.test('mark channel as fetched when messages are loaded', async function (assert) {
+    assert.expect(4);
+
+    this.data['res.partner'].records.push({
+        email: "fred@example.com",
+        id: 10,
+        name: "Fred",
+    });
+    this.data['mail.channel'].records.push({
+        channel_type: 'chat',
+        id: 100,
+        is_pinned: true,
+        members: [this.data.currentPartnerId, 10],
+    });
+    this.data['mail.message'].records.push({
+        body: 'not empty',
+        channel_ids: [ 100 ],
+    });
+    await this.start({
+        mockRPC(route, args) {
+            if (args.method === 'channel_fetched') {
+                assert.strictEqual(
+                    args.args[0][0],
+                    100,
```

Comment:
prefer asserting args[0] is (deep) equal to [100] (as array, to ensure there is no other element in it)